dist/split-test.js

```diff
@@ -25,6 +25,7 @@
         }
 
         this.cookieDomain = options.cookieDomain || window.location.hostname;
+        this.lifetime = options.lifetime || 0;
```

Comment:
Ah, it won't because if it's not greater then 0, we default to 30 seconds. Still though, you have to look deep in the code for that. I feel like it would be better to set this to the 30 day default right here so that it's less confusing for developers who look at this code later.